CONTRIBUTING.md

```diff
@@ -60,6 +60,25 @@ There are some additional style considerations that we do not have automated enf
 5. Small, incremental contributions are always preferred over sweeping changes. You should expect that any large sweeping change will be rejected summarily without review.
    If you're interested in working on something that _requires_ such a change, you should open an issue first to discuss the idea and get buy-in from the team.
 
+## Pull Request Guidelines
+
+When submitting a pull request:
+
+1. **PR Title:** Your PR title should be a clear, one-line sentence that makes sense as a changelog entry. This title will appear in the release notes, so write it to be informative to users. Focus on what changed from a user's perspective.
+   - Good: "Add support for custom error handlers in the CLI"
+   - Good: "Fix memory leak in filesystem operations"
```

Comment:
```suggestion
   - Good: "Adds support for custom error handlers in the CLI"
   - Good: "Fixes memory leak in filesystem operations"
```

If you want the titles to be changelog entries, they should be active.

Reply:
yup, just fixed.